tensilelite/Tensile/Tests/common/gemm/fp8fp16mix_hhs.yaml

```diff
@@ -0,0 +1,650 @@
+TestParameters:
+  marks: [skip-gfx900, skip-gfx906, skip-gfx908, skip-gfx90a, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030, skip-gfx1100, skip-gfx1101, skip-gfx1102] # not supported by arch
+
+GlobalParameters:
+  NumElementsToValidate: -1
+  MinimumRequiredVersion: 4.14.0
+  PrintLevel: 1
+  PrintSolutionRejectionReason: True
+  Device: 0
+  CMakeBuildType: Release
+  MergeFiles: False
+  KernelTime: True
+  MaxWorkspaceSize: 13421772800
+  DataInitTypeAlpha: 1
+  DataInitTypeBeta: 0
+  NumElementsToValidate: -1
+  BoundsCheck: 2
+
+BenchmarkProblems:
+  ########################################
+  # FP8FP16_HHS GlobalReadVectorWidthA
+  ########################################
+  -
+    - # ProblemType
+      OperationType: GEMM
+      DataType: h
+      DataTypeA: F8
```

Comment:
Can we use DataTypeA, DatatypeB?

Reply:
If DataTypeA/B is not specified, it'll fallback to DataType. The test makes sure this works properly.

Reply:
> Can we use DataTypeA, DatatypeB?

"Datatype" is still required since it is used to map to which mfma inst we would use.